tests/kernel/interrupt/testcase.yaml

```diff
@@ -11,3 +11,4 @@ tests:
     filter: not CONFIG_TRUSTED_EXECUTION_NONSECURE
     extra_configs:
       - CONFIG_CMAKE_LINKER_GENERATOR=y
+      - CONFIG_ISR_SHARE_IRQ=y
```

Comment:
Why are you testing this only for the `linker_generator` subtest? This should probably go in the `prj.conf` file instead if you want to test both cases (`CONFIG_CMAKE_LINKER_GENERATOR` and `!CONFIG_CMAKE_LINKER_GENERATOR`).

Reply:
Moved to prj.conf